scripts/winesapos-setup.sh

```diff
@@ -309,6 +309,47 @@ if [ $? -eq 0 ]; then
     sudo swapon /swap/swapfile
     echo "/swap/swapfile    none    swap    defaults    0 0" | sudo tee -a /etc/fstab
     qdbus ${kdialog_dbus} /ProgressDialog org.kde.kdialog.ProgressDialog.close
+elif [ $? -eq 1 ]; then
+    # If the user does not want swap, ask them about zram instead.
+    kdialog --title "winesapOS First-Time Setup" --yesno "Do you want to setup zram instead (recommended if you don't use swap)?"
```

Comment:
Minor nitpick: we try to avoid contractions. Use `do not` instead of `don't` as it is easier to read and understand.

Reply:
Updated that